packages/signers/src/signable-message.ts

```diff
@@ -0,0 +1,17 @@
+import { TextEncoder } from 'text-encoding-impl';
+
+import { SignatureDictionary } from './types';
+
+/** Defines a message that needs signing and its current set of signatures if any. */
+export type SignableMessage = {
+    content: Uint8Array;
+    signatures: SignatureDictionary;
+};
```

Comment:
Readonly, everywhere.

```suggestion
export type SignableMessage = Readonly<{
    content: Uint8Array;
    signatures: SignatureDictionary;
}>;
```